<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MepaGuiUi</class>
 <widget class="QDialog" name="MepaGuiUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MEPA BOGA</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Program Counter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Memory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Registers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="lPC"/>
     </item>
     <item>
      <widget class="QListWidget" name="lMEM">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lREG">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bNext">
       <property name="text">
        <string>Next Instruction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRun">
       <property name="text">
        <string>Run Program</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bNext</tabstop>
  <tabstop>bRun</tabstop>
  <tabstop>bQuit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bQuit</sender>
   <signal>clicked()</signal>
   <receiver>MepaGuiUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>runProgram()</signal>
 </slots>
</ui>
